gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0-all.zip
```

Comment:
You might want to run `gradlew wrapper` again. This version doesn't match the version in `build.gradle`.

Reply:
Good catch - I actually had crashes on 6.5 so I downgraded on the fly, forgetting to do so in build.gradle too. Will see if the problems persist and will either use 6.0 or 6.5 depending on which.

Reply:
Yup, problem persists on 6.5:

```
Unable to find a usable idle daemon. I have connected to 100 different daemons but I could not use any of them to run the build. BuildActionParameters were DefaultBuildActionParameters{, currentDir=/Users/anderseknert/git/opa-gradle-plugin, systemProperties size=57, envVariables size=47, logLevel=LIFECYCLE, useDaemon=true, continuous=false, injectedPluginClasspath=[]}.
```
"I have connected to 100 different daemons but I could not use any of them to run the build", haha! Can't blame them for not trying, that's for sure.

Reply:
> I have connected to 100 different daemons but I could not use any of them
Huh, that's my last experience with Apple's customer support.

Reply:
Lol yeah I just casually browsed LinkedIn, and I would say that description is just perfect for that network as well.